edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -93,7 +94,8 @@ class JwtAuthCookieMiddleware(object):
 
     Also, sets the metric 'request_jwt_cookie' with one of the following values:
         'yes': Value when reconstitution is successful.
-        'no': Value when both cookies are missing and reconstitution is not possible.
+        'no': Value when jwt cookie authentication was not requested by the client.
```

Comment:
`yes` and `no` aren't really opposites.   Should we use `success` and `not-requested`?

Reply:
Sounds good. I will update.